sidebars/index.js

```diff
@@ -1,7 +1,15 @@
-const Home = require('./home');
 const Core = require('./core');
+const Home = require('./home');
```

Comment:
Before, when clicking on any section besides core contributors or reference, there would be no corresponding menu. This adds the menus according to the comps

Reply:
<img width="302" alt="Screen Shot 2020-06-14 at 11 26 17 PM" src="https://user-images.githubusercontent.com/20331238/84620864-5570ba80-ae96-11ea-9152-34713e1356cd.png">